config/environments/development.rb

```diff
@@ -14,7 +14,7 @@
 
   # Enable/disable caching. By default caching is disabled.
   # Run rails dev:cache to toggle caching.
-  if Rails.root.join('tmp', 'caching-dev.txt').exist?
+  if Rails.root.join('/tmp', 'caching-dev.txt').exist?
```

Comment:
Actually, this is only one place where `/tmp` is referenced. And also, `Rails.root` is wherever the the app dir is, so this change should have no effect really.

Reply:
Thanks for catching that!